Scripts/Python/hopsan.py

```diff
@@ -0,0 +1,52 @@
+class hopsan:
+    def __init__(self):
+        import ctypes
+        import os
+        self.hdll = ctypes.cdll.LoadLibrary(os.getenv('HOPSANHOME')+"/bin/libhopsanc.so")
+        libpath= os.getenv('HOPSANHOME')+"componentLibraries/defaultLibrary/libdefaultcomponentlibrary.so"
+        self.hdll.loadLibrary(libpath.encode())
+
+    def loadModel(self, path):
+        self.hdll.loadModel(path.encode())
+
+    def simulate(self):
+        self.hdll.simulate()
+
+    def getTimeVector(self):
+        samples = self.hdll.getLogSamples()
```

Comment:
Hmm, I think it would be more clear, if the c-api function was called getNumberOfLogSamples, or similar. Without thinking I thought that this coped all of the samples logged. If you change it, remember the julia code.

Reply:
I changed it in the C  API for both get and set functions. I decided to keep the `setLogSamples` name in the Python and Julia APIs, to keep it simple.